recipes/correct-ts-specifiers/src/exts.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * A map of JavaScript file extensions to the corresponding TypeScript file extension.
+ */
+export const jsToTSExts = {
+	'.cjs': '.cts',
+	'.mjs': '.mts',
+	'.js': '.ts',
+	'.jsx': '.tsx',
+} as const;
```

Comment:
```suggestion
} as const;

```

Give little bit of "air" to the code

Reply:
Ah, we should add this as a biome rule

Reply:
@AugustinMauroy I couldn't find such a rule for biome. Do you know if it exists? If so, could you please add it in https://github.com/nodejs/userland-migrations/pull/6

Reply:
I'll read for that, instead ask them